src/index.html

```diff
@@ -6,9 +6,29 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#"><img src="./images/logo.png" alt="MOYO Logo"></a>
+      <nav class="nav">
+        <ul class="categories">
+          <li class="button"><a class="link is-active" href="#">Apple</a></li>
+          <li class="button"><a class="link" href="#">Samsung</a></li>
+          <li class="button"><a class="link" href="#">Smartphones</a></li>
+          <li class="button"> <a class="link" data-qa="hover"
+                href="#">Laptops & Computers</a>
```

Comment:
```suggestion
          <li class="button">
            <a
              class="link"
              data-qa="hover"
              href="#"
            >
              Laptops & Computers
            </a>
```

Reply:
Pay attention to formatting. Fix similar problems in all places
<img width="1052" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/86789161/34494e59-6f63-45bd-b8d4-aef3640ed298">

Reply:
Sir, yes, sir! I'm already working on it!)
Do I need to follow the same pattern inside the head tag as well?